code/data_exploration.ipynb

```diff
@@ -873,6 +873,437 @@
     "data_pheno.columns"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "6ddf7e5d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    1\n",
+       "1    0\n",
+       "2    1\n",
+       "3    0\n",
+       "4    1\n",
+       "5    0\n",
+       "6    1\n",
+       "7    0\n",
+       "8    1\n",
+       "9    0\n",
+       "Name: adhd, dtype: int64"
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# check column 'adhd'\n",
+    "data_pheno['adhd'].head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "937a7986",
+   "metadata": {},
+   "source": [
+    "As we can see in the list of columns, the phenotypic file loaded here, does not contain a clustering of the participants regarding their diagnosis (ADHD-combined, ADHD-inattentive or ADHD_hyperactive/impulsive). But the phenotypic file of the whole ADHD-200 sample seems to hold this information in a column names 'diagnosis', as we can check [here](http://fcon_1000.projects.nitrc.org/indi/adhd200/general/ADHD-200_PhenotypicKey.pdf).\n",
+    "\n",
+    "So let's load the phenotypic file of the whole dataset. We can determine the diagnosis later using the subject id."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "dba67cde",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "('/mnt/c/Users/nasta/Desktop/Forschungsmodul/project_msc05/data/adhd200_preprocessed_phenotypics.tsv',\n",
+       " <http.client.HTTPMessage at 0x7f78cd237320>)"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# load phenotypic file of the whole ADHD-200 sample to explore it\n",
```

Comment:
Comments like such can be converted/copied into `markdown cells`, makes it easier to read and follow.